pages/treeherder.py

```diff
@@ -122,6 +122,10 @@ def unclassified_failure_count(self):
     def clear_filter(self):
         self.selenium.find_element(*self._clear_filter_locator).click()
 
+    def clear_filter_by_shortcut(self):
```

Comment:
I would maybe propose a `method` argument to the existing method, which defaults to 'pointer' but can also be 'keyboard'. What do you think?